conf/defaults.ini

```diff
@@ -458,7 +458,8 @@ url = https://grafana.com
 provider =
 
 [external_image_storage.s3]
-bucket_url =
```

Comment:
this needs to be added back, as the keys defined in defaults are used to know what keys can be set (and what ENV variables to look for, so it it is not here you cannot override it from command line or ENV variable)

Reply:
since it was such a minor fix I merged and did this myself, thanks for contributing!